tests/mt/forward/test_Problem1D_AnalyticVsNumeric.py

```diff
@@ -1,76 +1,37 @@
 import unittest
 import SimPEG as simpeg
-from SimPEG import MT
+from SimPEG import NSEM
 from SimPEG.Utils import meshTensor
 import numpy as np
 # Define the tolerances
 TOLr = 5e-2
 TOLp = 5e-2
 
 
-def setupSurvey(sigmaHalf,tD=True):
-
-    # Frequency
-    nFreq = 33
-    freqs = np.logspace(3,-3,nFreq)
-    # Make the mesh
-    ct = 5
-    air = meshTensor([(ct,25,1.3)])
-    # coreT0 = meshTensor([(ct,15,1.2)])
-    # coreT1 = np.kron(meshTensor([(coreT0[-1],15,1.3)]),np.ones((7,)))
-    core = np.concatenate( (  np.kron(meshTensor([(ct,15,-1.2)]),np.ones((10,))) , meshTensor([(ct,20)]) ) )
-    bot = meshTensor([(core[0],15,-1.3)])
-    x0 = -np.array([np.sum(np.concatenate((core,bot)))])
-    m1d = simpeg.Mesh.TensorMesh([np.concatenate((bot,core,air))], x0=x0)
-    # Make the model
-    sigma = np.zeros(m1d.nC) + sigmaHalf
-    sigma[m1d.gridCC > 0 ] = 1e-8
-    sigmaBack = sigma.copy()
-    # Add structure
-    shallow = (m1d.gridCC < -200) * (m1d.gridCC > -600)
-    deep = (m1d.gridCC < -3000) * (m1d.gridCC > -5000)
-    sigma[shallow] = 1
-    sigma[deep] = 0.1
-
-    rxList = []
-    for rxType in ['z1dr','z1di']:
-        rxList.append(MT.Rx(simpeg.mkvc(np.array([0.0]),2).T,rxType))
-    # Source list
-    srcList =[]
-    if tD:
-        for freq in freqs:
-            srcList.append(MT.SrcMT.polxy_1DhomotD(rxList,freq))
-    else:
-        for freq in freqs:
-            srcList.append(MT.SrcMT.polxy_1Dprimary(rxList,freq))
-
-    survey = MT.Survey(srcList)
-    return survey, sigma, m1d
-
-def getAppResPhs(MTdata):
+def getAppResPhs(NSEMdata):
```

Comment:
why the 8e-7 ? are there simplifications that are happening? If so, I would suggest keeping the mu_0, and perhaps even consider it as a kwarg input `mu=mu_0` - this keeps the assumptions more transparent